config/initializers/geocoder.rb

```diff
@@ -21,6 +21,7 @@
 #)
 
 require Rails.root.join('lib', 'geocoder_autoexpire_cache')
+require File.join('geocoder', 'lookups', 'bing')
```

Comment:
This line shouldn't be required now - since you're referencing Geocoders version of `Geocoder::Lookup::Bing`, it should be found via the autoloader? 

It would only have been required to avoid your own declaration of the class in the previous iteration preventing autoloading.

Reply:
I needed this to make it work, without it it failed due to it not having yet been loaded. In the earlier version, this line wasn't present.

You're welcome to have a look, I couldn't make it work without the `require`

Reply:
Thats fine - now I've taken a look at the geocoder I can see rather than autoloading, it does the require in `Lookup.spawn` - which is after we're wanting to `prepend` so the `require` is fine.

Thanks for looking at this